actions/ldap

```diff
@@ -92,13 +92,29 @@ get_user_groups()
 }
 
 
+add_group()
+{
+    groupname="$1"
+
+    ldapsearch -Q -L -L -L -Y EXTERNAL -H ldapi:/// -s base -b "cn=${groupname},dc=thisbox" || ldapaddgroup "${groupname}" > /dev/null 2>&1
```

Comment:
I tested this by installing Deluge and also running this directly on the command line. 

```
root@freedombox:/usr/share/plinth/actions# ./ldap add-group bittorrent
No such object (32)
Matched DN: dc=thisbox
```

The user group bittorrent wasn't being created.

```
root@freedombox:/usr/share/plinth/actions# cut -d: -f1 /etc/group | sort | head
adm
audio
avahi
backup
bin
cdrom
crontab
daemon
debian-deluged
debian-transmission
```

Reply:
@JosephKiranBabu I don't think the group will be listed in /etc/group, since it is stored in ldap instead. I think you can use "getent group" to check it.

Reply:
Sorry, my bad. `bittorrent` group was created in ldap. Will merge.